### DISCUSSION TITLE
external Keyboard distinguishable?

### DISCUSSION THREAD

**Discussion by pictosun** [rxhanson/Multitouch-Community#142] [Q&A]
Hi,

is there any chance to get an external Keyboard support? So that the remapped shortcuts do behave on keyboard basis (internal/external)?

Why? Because the key mapping of my Logitech keyboard is different than the mapping of my original keyboard.

**Comment by rxhanson** [comment]
Thanks for the feedback. I do have plans for this, but getting it right in the UI can be difficult. It's kind of a question of how customizable I want to make it. One simple way is to have one checkbox for non-apple keyboards, is that something that you would be after?

**Reply by pictosun** [reply]
Yes. As it's an easy to use app and it should stay that way. 

I think a simple check-box for non-apple or external keyboards should do it all. 

And what I do find important is to have the option the remap the 'FN' key, as it's often wrong positioned on external/non-apple keyboards. A remap of this button is a night and day feature.

**Reply by rxhanson** [reply]
The reason I haven't added in fn key remapping is because it doesn't work like the rest of the keys. On my Logitech ergonomic keyboard, the fn key doesn't actually trigger fn in macOS, it merely allows you to execute the alternate set of fn key functionality on the function keys. Remapping fn would not actually remap anything on my Logitech keyboard, although it would remap it on my Mac keyboard.

If you don't have the fn key remap, is this feature still valuable to you?

**Reply by pictosun** [reply]
Thx for the feedback. Main Point would really be this stupid FN button. Don't know why many keyboards do the flip of Control <> FN keys. 
If you're working with different brands it's so annoying. That would actually be my main point of interest. Yes.
